tools/bionano/bionano_scaffold_edited.xml

```diff
@@ -91,7 +99,7 @@
         <flag attr="hashgen" val0="5" val1="7" val2="2.4" val3="1.5" val4="0.05" val5="5.0" val6="1" val7="1" val8="4"/>
         <flag attr="hash" val0="-hashdelta" val1="26" val2="10" val3="46"/>
         <flag attr="hashMultiMatch" val0="30" val1="10"/>
-        <flag attr="insertThreads" val0="$cores"/>
+        <flag attr="insertThreads" val0="54"/>
```

Comment:
why do you revert this?

Reply:
Yes, sorry, It was a fail. I changed all the configuration file because the one provided in the test correspond to the samples used on it. I'll fix it.

Reply:
I didn't realize that there were more parameters that required to be modified.

Reply:
There are a few more ...